created 3 classes

GraduateRam opened the pull request:
I categorized the broad group of fungal infectious diseases into three distinct classes
i)subcutaneous fungal infectious disease,
ii)superficial fungal infectious disease,
iii) systemic fungal infectious disease.

GraduateRam committed:
created 3 classes

created 3 classes for fungal infectious disease
i)subcutaneous fungal infectious disease,
ii)superficial fungal infectious disease,
iii) systemic fungal infectious disease.

GraduateRam committed:
multiple changes

-imported IDO.owl
-fixed logical reasoning errors
-fixed duplicate definition errors
-changed 'has definition' to 'definition'
-added term editor annotations

GraduateRam commented:
@giacomodecolle Added u in the comment

giacomodecolle commented on mido.ttl:
```diff
@@ -8794,42 +8782,44 @@ obo:NCIT_C162141 rdf:type owl:Class ;
                  obo:IAO_0000412 "http://purl.obolibrary.org/obo/NCIT" ;
-                 oboInOwl:hasDefinition "A Cryptococcus that affects the membranes surrounding the brain and spinal column (meninges). It is a blood borne fungal infection caused by Cryptococcus, usually C. neoformans or C. gattii. It usually occurs in immunocompromised patients including persons with AIDS, transplant recipients, patients receiving cytotoxic chemotherapy, and patients with hemato ...
-                                        "A cryptococcosis that is characterized by a blood borne fungal infection of the membranes surrounding the brain and spinal column (meninges). Caused by Cryptococcus, usually C. neoformans or C. gattii. It usually occurs in immunocompromised patients including persons with AIDS, transplant recipients, patients receiving cytotoxic chemotherapy, and patients  ...
+                 oboInOwl:hasDefinition "A Cryptococcus that affects the membranes surrounding the brain and spinal column (meninges). It is a blood borne fungal infection caused by Cryptococcus, usually C. neoformans or C. gattii. It usually occurs in immunocompromised patients including persons with AIDS, transplant recipients, patients receiving cytotoxic chemotherapy, and patients with hemato ...
```
Are these two definitions for the same class?

GraduateRam replied:
Yeah! Both of the definitions refer to the same class. I didn't work on this class. I believe I should eliminate the second definition because it differs from the NCIT-related one.
![Image 04-11-24 at 11 22 AM](https://github.com/user-attachments/assets/2dd4b7d6-0713-4eef-9cae-92723db72a52)

giacomodecolle commented on mido.ttl:
```diff
@@ -8794,42 +8782,44 @@ obo:NCIT_C162141 rdf:type owl:Class ;
-                                                                                   ]
-                                                                                 ) ;
-                                                              rdf:type owl:Class
+                 owl:equivalentClass [ owl:intersectionOf ( obo:NCIT_C2886
```
Are we sure we should be changing axioms for NCIT?

giacomodecolle commented on mido.ttl:
```diff
@@ -8794,42 +8782,44 @@ obo:NCIT_C162141 rdf:type owl:Class ;
+                 obo:IAO_0000117 "https://orcid.org/0009-0003-7246-1252 \"Ram A.N.R. Challa\"" ;
                  obo:IAO_0000412 "http://purl.obolibrary.org/obo/NCIT" ;
-                 oboInOwl:hasDefinition "An aspergillosis resulting in a chronic infectious disease course affecting the lungs." ;
                  rdfs:comment "Occurs when Aspergillus infection causes cavities in the lungs, and can be a long-term (3 months or more) condition. One or more fungal balls (aspergillomas) may also be present in the lungs.  It is characterized by the slow progression of lung damage over months to years and is associated with symptoms such as chronic cough, fatigue, weight loss, and shortness of b ...
```
Are you taking away a definition from an NCIT class?

giacomodecolle commented on mido.ttl:
```diff
@@ -8846,7 +8836,8 @@ obo:NCIT_C28266 rdf:type owl:Class ;
 obo:NCIT_C2886 rdf:type owl:Class ;
                rdfs:subClassOf <http://www.co-ode.org/ontologies/ont.owl#MIDO_00111> ;
-               oboInOwl:hasDefinition "An opportunistic fungal infectious disease caused by fungi of the Aspergillus species that affects the skin, ears, nasal sinuses, and lungs of people with suppressed immune systems." ;
+               obo:IAO_0000115 "An opportunistic fungal infectious disease caused by fungi of the Aspergillus species that affects the skin, ears, nasal sinuses, and lungs of people with suppressed immune systems." ;
```
Same question, should we be changing definitions of NCIT classes?

giacomodecolle commented on mido.ttl:
```diff
@@ -8872,22 +8863,24 @@ obo:NCIT_C3245 rdf:type owl:Class ;
 ###  http://purl.obolibrary.org/obo/NCIT_C34891
 obo:NCIT_C34891 rdf:type owl:Class ;
                 rdfs:subClassOf <http://www.co-ode.org/ontologies/ont.owl#MIDO_00112> ;
+                obo:IAO_0000117 "https://orcid.org/0009-0003-7246-1252 \"Ram A.N.R. Challa\"" ;
```
Are you adding yourself as creator for one of the classes of NCIT?

giacomodecolle commented on mido.ttl:
```diff
@@ -8872,22 +8863,24 @@ obo:NCIT_C3245 rdf:type owl:Class ;
 ###  http://purl.obolibrary.org/obo/NCIT_C4985
 obo:NCIT_C4985 rdf:type owl:Class ;
-               rdfs:subClassOf obo:IDO_0000645 ;
+               rdfs:subClassOf <http://www.co-ode.org/ontologies/ont.owl#candidiasis_disease> ;
```
Same question about modifying classes in NCIT

giacomodecolle commented on mido.ttl:
```diff
@@ -9913,6 +9889,7 @@ doi: 10.3904/kjim.2004.19.1.38""" ;
 <http://www.co-ode.org/ontologies/ont.owl#MIDO-00114> rdf:type owl:Class ;
                                                       rdfs:subClassOf <http://www.ebi.ac.uk/efo/EFO_0007174> ;
                                                       obo:IAO_0000115 "A blastomycosis that is caused by Blastomyces dermatitidis, primarily affecting the skin." ;
+                                                      obo:IAO_0000117 "https://orcid.org/0009-0003-7246-1252 \"Ram A.N.R. Challa\"" ;
```
Are these the classes you worked on previously?

GraduateRam committed:
Overall changes

•	The base URI of all terms has been changed from 'http://www.co-ode.org/ontologies/' to 'http://purl.obolibrary.org/obo/' in order to keep things the same and standardized. I did this to use the PURL given by the OBO Foundry.
•	Definitions are slightly deviated to be closer to the parent class terms. Hence, I replaced their prefixes, such as NCIT, EFO with MIDO.
•	The old prefixes of five terms, including cutaneous aspergillosis, invasive aspergillosis, cryptococcal meningitis, endophthalmitis, and allergic bronchopulmonary aspergillosis, remain unchanged because these terms were not altered as there is no deviation from their parent classes.
•	Added 'Definition sources’.

johnbeve commented:
Is this PR stale or does it still need to be reviewed?
